installation_tips/full_spikeinterface_environment_windows.yml

```diff
@@ -25,10 +27,10 @@ dependencies:
   - ipympl
   - pip:
     - ephyviewer
-    - neo>=0.12
-    - probeinterface>=0.2.11
-    - MEArec>=1.8
-    - spikeinterface[full, widgets]
+    - neo>=0.13
+    - probeinterface
```

Comment:
Why drop the limit? Assuming that only new users will install?

Reply:
Normaly the limit is done by the project.toml

Reply:
This could even be removed.

Reply:
Then why neo?

Reply:
Same should eb removed